public/electron.js

```diff
@@ -40,12 +40,62 @@ function createWindow() {
 // Menu
 const isMac = process.platform === 'darwin'
 const template = [
+    {
+        label: "Application",
+        submenu: [
+            { 
+                label: "Quit", 
+                accelerator: "Command+Q", 
```

Comment:
Shouldn't this be `CmdOrCtrl+Q`, so this hotkey also works on Windows?

Reply:
I think alt+f4 is more commonly used, I cannot test it because I don't have a windows pc

Reply:
`ALT+F4` is a Windows OS key combination to shut down an application, but `CTRL+Q` is also a common key combination that applications implement themselves.